UNRELEASED.md

```diff
@@ -1 +1,12 @@
 # Unreleased
+
+**Added:**
+
+- react-native-bpk-theming:
+  - Added a new HOC `withPrimaryColor` to support arbitrary usage of a theme's `primaryColor` attribute. If rendered outside of a `BpkThemeProvider` with a defined theme `colorBlue500` is used instead.
+    ```javascript
+    const Color = withPrimaryColor((props) => (
+      <View style={{ backgroundColor: props.primaryColor }} />
+    ));
```

Comment:
Having a code sample is a great move - we should do this more in other PRs really!